docs/install.rst

```diff
@@ -30,63 +29,119 @@ The easiest way to install Python and SciPy is with `Anaconda`_, a free scientif
 
 .. rubric:: Windows
 
-#. Go to the `Anaconda`_ website and download the Python 3.6 installer.
+#. Go to the `Anaconda`_ website and download the Python 3.8 (or newer) installer.
 
 #. Run it and accept the default options during the installation.
 
-#. Open `Anaconda Prompt` from the `Start` menu. Enter the following command a press enter. This will enable the `conda-forge`::
+#. Open `Anaconda Prompt` from the `Start` menu.
+
+#. Create a new environment for Pylake named pylake by invoking the following command::
+
+    conda create -n pylake
+
+#. The environment can then be activated::
+
+    conda activate pylake
+
+#. Verify that `(pylake)` is now prefixed to the path in your anaconda prompt (this means the environment is activated).
+
+#. Pylake can be found on `conda-forge`_. Add `conda-forge`_ to the list of sources as follows::
 
     conda config --add channels conda-forge
 
-#. Finally, enter the following command to install Pylake::
+#. We can install Pylake in this environment by invoking the following commands::
 
     conda install lumicks.pylake
 
+#. To make sure Jupyter notebook can find the ipykernel, we invoke::
+
+    python -m ipykernel install --user
+
+#. It should be possible to open a jupyter notebook in this environment by calling::
+
+    jupyter notebook
```

Comment:
These are useful steps! I didn't know. I always end up installing the Notebook in each and every environment, but this is much better 😄

Reply:
👍